pages/project-details/[id]/view-details/[viewId].js

```diff
@@ -53,10 +41,6 @@ export default function ViewDetails({ views }) {
   );
 }
 
-const StyledImage = styled(Image)`
-  object-fit: cover;
-`;
-
 const DescriptionText = styled.p`
   overflow: scroll;
```

Comment:
Same as before.
You kind of get ugly scrollbars in desktop view. 

![Screenshot 2023-01-29 at 10 31 49](https://user-images.githubusercontent.com/117607177/215317665-b7f85742-0da4-4aa8-b8ab-f8df8d361672.png)